models/wholeBody_ct_segmentation/docs/README.md

```diff
@@ -32,13 +32,28 @@ The segmentation of 104 tissues is formulated as voxel-wise multi-label segmenta
 
 The training was performed with the following:
 
-- GPU: 32 GB of GPU memory
+- GPU: 48 GB of GPU memory
 - Actual Model Input: 96 x 96 x 96
 - AMP: True
 - Optimizer: AdamW
 - Learning Rate: 1e-4
 - Loss: DiceCELoss
 
+### GPU consumption warning:
+
+The model is trained with 104 classes in single instance, for predicting 104 structures, the GPU consumption can be large.
+
+For inference pipeline, please refer to the following section for benchmarking results. Normally, a CT scans with 300 slices will take about 27G memory, if your CT is larger, please prepare larger GPU memory or use CPU for inference.
+
+
+### RAM consumption warning:
```

Comment:
Here is my suggestion:
"### Memory Consumption
- Dataset Manager: CacheDataset
- Data Size: 1000 3D Volumes
- Cache Rate: 0.4
- Single GPU VRAM: 83G
- Multi GPU VRAM: 666G (8 GPUs)
"